cmd/tealdbg/debugger_test.go

```diff
@@ -90,6 +90,7 @@ func (d *testDbgAdapter) eventLoop() {
 
 func TestDebuggerSimple(t *testing.T) {
 	proto := config.Consensus[protocol.ConsensusV23]
+	require.False(t, proto.Application)
```

Comment:
it does not matter here

Reply:
Would it be correct to use "ConsensusCurrentVersion" in here?

Reply:
or maybe I should ask : why version 23 was picked here ?

Reply:
Anything with LogicSig enabled, so `protocol.ConsensusV23` is just fine. I do not like `ConsensusCurrentVersion` since we might want disabling LogicSig one day.

Reply:
Because I was lazy to find a version where LogicSig was enabled, and `protocol.ConsensusV23` is known to be working.

Reply:
LogicSigVersion > 1 ? That was v18. LogicSigVersion > 2 ? that's V24..

Reply:
ok, I updated it to use the v18 and LogicSigVersion > 0 as a predicate.